rest/rest-server/src/main/java/org/apache/brooklyn/rest/util/DefaultExceptionMapper.java

```diff
@@ -92,13 +93,17 @@ public Response toResponse(Throwable throwable1) {
                 LOG.warn("REST call generated exception type "+throwable2.getClass()+" unrecognized in "+getClass()+" (subsequent occurrences will be logged debug only): " + throwable2, throwable2);
             }
         }
+
+        // Before saying server error, look for a user-facing exception anywhere in the hierarchy
```

Comment:
Should this not be `Exceptions.getFirstThrowableOfType(e, UserFacingException.class)` if we're looking for it anywhere in the hierarchy?